src/pages/Calendar/Caption/caption.style.ts

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components/native";
+import { ColorScheme } from "@styles/globalStyles";
+import { FontScheme } from "@styles/globalStyles";
+
+export const Sc = {
+  Captions: styled.View`
+    flex-direction: row;
+    gap: 96px;
+  `,
+
+  LeftCaptions: styled.View`
+    flex-direction: column;
+  `,
+
+  RightCaptions: styled.View`
+    flex-direction: column;
+  `,
+
+  GroupCaptions: styled.View`
+    flex-direction: row;
+    margin-bottom: 13px;
+    gap: 12px;
+  `,
+
+  Container: styled.View`
+    width: 358px;
+    height: 112px;
+    margin: 0 auto;
+    background-color: ${ColorScheme.white.primary};
+    justify-content: center;
+    align-items: center;
+    padding: 16px;
+    gap: 11px;
+  `,
+
+  Title: styled.Text`
+    font-size: 16px;
```

Comment:
For font sizes we have it all set in the ${FontScheme.size}, in this case it would be the "default" which equals 16px